packages/ListBox/stories/examples/single.js

```diff
@@ -23,6 +23,15 @@ export const BasicWithEmptyOption = () => (
   </ListBox>
 );
 
+export const BasicWithImplicitAll = () => (
+  <ListBox placeholder="All">
+    <ListBox.Trigger hasImplicitAll />
```

Comment:
If the `placeholder` prop is applied to the `ListBox` root component, I feel like the `hasImplicitAll` prop should also be there.